ui/app/AppLayouts/Wallet/popups/dapps/DAppsWorkflow.qml

```diff
@@ -77,6 +83,9 @@ SQUtils.QObject {
 
     property var formatBigNumber: (number, symbol, noSymbolOption) => console.error("formatBigNumber not set")
 
+    property bool walletConnectEnabled: true
+    property bool connectorEnabled: true
```

Comment:
Unconditionally on now?

Reply:
the default is true. But the proper value is set in the `AppMain`, depending on the feature flags.